src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,45 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class HomePage extends BasePage {
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+//    public HomePage(WebDriver driver) {
+//        super(driver);
+//    }
+
+    public void clickOnPlaylist(String playlistName) {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//a[contains(text(),'" + playlistName + "')]"))).click();
+    }
+
+    public void clickAllSongs() {
+        wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("li a.songs"))).click();
+    }
+    public void clickPlay() {
+        wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector(".playback"))).click();
+    }
+    public  void checkSongIsPlaying() {
+        WebElement soundBar = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[data-testid='sound-bar-play']")));
+        Assert.assertTrue(soundBar.isDisplayed());
+    }
+    public void checkIfPlayBtnIsVisible() {
+        WebElement playBtn =   wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[data-testid='play-btn']")));
+        Assert.assertTrue(playBtn.isDisplayed());
+    }
+
+    public void mouseMoveToPlayBtn(){
+        WebElement playPanel = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".side.player-controls")));
+        actions.moveToElement(playPanel).click(playPanel).perform();
```

Comment:
We can make the xpath or css the class variable (Locator)

For example

![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/d327eccd-5b31-455b-b586-1bbad7d539d5)